src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -904,16 +904,12 @@ ValidatorList::applyListsAndBroadcast(
     bool broadcast = disposition <= ListDisposition::known_sequence;
 
     // localPublisherList is not broadcast
-    if (broadcast)
+    if (broadcast && result.status <= PublisherStatus::expired &&
+        result.publisherKey &&
+        publisherLists_[*result.publisherKey].maxSequence)
     {
-        assert(
-            result.status <= PublisherStatus::expired && result.publisherKey &&
-            publisherLists_[*result.publisherKey].maxSequence);
         auto const& pubCollection = publisherLists_[*result.publisherKey];
 
-        // The below function call is executed for PublicKeys which are not
-        // specified in the config file. The assert statement above filters
-        // out PublisherLists which are not associated with a master key.
```

Comment:
I suggest a comment just before or after the `if` that's something like:
```
// broadcastBlobs is only called for PublicKeys which are not specified in the config file.
```

Reply:
ok